src/app/components/Speakers.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import config from '../config.json'
+
+class Speakers extends Component {
+  shouldRender (edition) {
+    return config[edition].indexOf('Speakers') > -1
+  }
+  render () {
+    return (
+      <div>
+        { this.shouldRender(this.props.edition) && <h3> The speakers component is rendered</h3> }
```

Comment:
Yes, I think it can. I discussed it with @JGAntunes, we were not 100% sure which way is better but we concluded yours might be simpler. I'll update the PR.